settings.gradle

```diff
@@ -19,3 +23,25 @@ includeBuild("../jdk") {
 includeBuild("../checker-framework") {
     initializeProject()
 }
+
+dependencyResolutionManagement {
+    versionCatalogs {
+        libs {
+            version("checkerFramework", "3.21.5-SNAPSHOT")
+
+            library("checkerFramework-checker", "org.checkerframework", "checker").versionRef("checkerFramework")
```

Comment:
I was confused why here the libraries use `-`, but then in the uses in `build.gradle` the libraries use `.`.
https://docs.gradle.org/current/userguide/platforms.html#sub:version-catalog-declaration explains this and the recommendation (for whatever reason) is to use `-` here, but then requires to use `.` in the uses...
Is it worth adding that manual URL here?

Reply:
I don't think it is worth it. This is how version catalogs work; it's not a choice we're making. I wouldn't expect any usages of features of Gradle to be documented.

The reason it's dots in  `build.gradle` is because these become objects with properties: `libs` has a property `checkerFramework` which has a property `checker`. That's Java/Groovy property notation, and identifiers can't have hyphens.